### PR TITLE
new Proposal System big merge

### PR DESCRIPTION
**Pull request by steinerkelvin**
by luiz

### CODE REVIEW

File: pallets/subspace/rpc/runtime-api/src/lib.rs [hunk 1]
```diff
@@ -17,8 +17,9 @@ pub type AccountId = <<Signature as Verify>::Signer as IdentifyAccount>::Account
 pub struct ModuleStats {
     pub last_update: u64,
     pub registration_block: u64,
-    pub stake_from: Vec<(AccountId, u64)>, /* map of key to stake on this module/key * (includes
-                                            * delegations) */
+    pub stake_from: BTreeMap<AccountId, u64>, /* map of key to stake on this module/key *
+                                               * (includes
+                                               * delegations) */
```
Comment (aripiprazole) [inline pallets/subspace/rpc/runtime-api/src/lib.rs:22]: use `///`

Comment (Supremesource) [reply pallets/subspace/rpc/runtime-api/src/lib.rs]: done

**Review by Supremesource — COMMENTED** [review]
fixed the failing test + migrations. Looking ready to get merged

**Review by steinerkelvin — COMMENTED** [review]
LGTM

**Pull request merged**